package.json

```diff
@@ -7,7 +7,10 @@
   "author": "ESPHome maintainers",
   "license": "Apache-2.0",
   "scripts": {
-    "prepublishOnly": "script/build"
+    "develop": "script/develop",
+    "build": "script/build",
```

Comment:
Why would we add these here? They are never used except for on CI?

Reply:
by CI maybe yes, but I use them from VS Code to start dev servers:
![image](https://user-images.githubusercontent.com/1741838/203252182-bfa7bec2-a2e8-4d90-a5b6-644f3c8c3608.png)

Reply:
You don’t ever call the build script though. 

Also, isn’t that provided by a plugin? I don’t have those.

Reply:
It is built-in functionality: https://code.visualstudio.com/updates/v1_23#_npm-script-running.
I've added all scripts because now you can use npm to run them.
In the past, I did some things in React, and by default when creating new projects all actions are added to the scripts section in package.json.